tests/snuba/sessions/test_sessions_v2.py

```diff
@@ -325,6 +325,40 @@ def test_massage_simple_timeseries():
     assert actual_result == expected_result
 
 
+@freeze_time("2020-12-18T11:14:17.105Z")
+def test_massage_unordered_timeseries():
+    """A timeseries is filled up when it only receives partial data"""
```

Comment:
is this docstring accurate?

Reply:
it should be accurate, but its a repeat from the test case above. I don't think its needed here so ill remove it